doc/en/user/source/styling/mbstyle/installing.rst

```diff
@@ -3,8 +3,14 @@
 Installing the GeoServer MBStyle extension
 ==========================================
 
- #. Download the extension from the `nightly GeoServer community module builds <https://build.geoserver.org/geoserver/master/community-latest/>`_.
+The MBStyle extension is listed on the GeoServer download page.
```

Comment:
An actual link to the download page might be nice here, since there was a link before.

Reply:
Trying to think what to do for that, either link to the geoserver release page, or straight to source forge.

Reply:
I'd say just the GeoServer release page would be simplest

Reply:
Actually that ended up being hard:

- We could hard code it (and then have a maintenance trouble updating each page)
- There was no way to use substitutions (since those cannot be used in generating links)
- Ended up making three external link definitions (artifacts, extension plugins, community plugins) ... and then making those dynamic based on "snapshot" (build server) or "release" (source forge)

Please have a look.

Reply:
You went way fancier then I was expecting, I was just thinking linking http://geoserver.org , or http://geoserver.org/release/{version} 
I was just looking for something of comparable utility to the prior link.

Your solution does seem reasonable, assuming it is stable - i.e. doesn't break next time the we update versions. It looks stable enough, but sometimes the release scripts miss a version # somewhere (or a step gets inadvertently skipped).

Reply:
Yeah as above there was no way to add "version" into a URL, so no kill like overkill.